packages/example/src/pages/CustomUserDrawerPage.tsx

```diff
@@ -45,20 +48,49 @@ const searchPlaceholder = "Search area names, etc.";
 
 
 const CustomUserDrawerPage : React.FC = () => {
+  const  [attributeLanguage, setAttributeLanguage] = useState('us');
+
+  const onLanguageToggle = useCallback(() => {
+    setAttributeLanguage((prev:  string) => {
+      const newLang = prev === 'en'? 'ja' : 'en'
+      return newLang;
+    })
+  },[])
 
 
   const userDrawerBespoke = <CustomContentContainer>
     <ExampleText>
-      <p>Custom content can be injected under the user menu like this.</p>
-      <p>The styling for this is left to you to include, keeping it flexible.</p>
+      {attributeLanguage === 'en' ?
```

Comment:
As we tend to do everything English first, and English becomes our fallback language by default, it would be good to swap the order here, so it's conditional on `ja` else it goes to the English.

Reply:
Thank you for the suggestion, I will update.